sys/netbsd/fs.txt

```diff
@@ -44,10 +45,29 @@ dup2(oldfd fd, newfd fd) fd
 dup3(oldfd fd, newfd fd, flags flags[dup_flags]) fd
 pipe2(pipefd ptr[out, pipefd], flags flags[pipe_flags])
 
+pathconf(file ptr[in, filename], name flags[conf_value])
+fpathconf(fd fd, name flags[conf_value])
+
 compat_40_mount(type ptr[in, string[filesystem_types]], path ptr[in, filename], flags flags[mount_flags], data buffer[in])
 __mount50(type ptr[in, string[filesystem_types]], path ptr[in, filename], flags flags[mount_flags], data buffer[in], len len[data])
 unmount(path ptr[in, filename], flags flags[mount_flags])
 
+compat_90_statvfs1(path ptr[in, filename], buf ptr[out, statvfs90], f flags[vfs_flags])
+compat_90_fstatvfs1(fd fd, buf ptr[out, statvfs90], f flags[vfs_flags])
+
+compat_30_getfh(path ptr[in, filename], fhp ptr[out, compat_30_fhandle])
+__getfh30(fname ptr[in, filename], fhp buffer[out], fh_size ptr[in, intptr])
+compat_30_fhopen(fhp ptr[in, compat_30_fhandle], f flags[fhopen_flags]) fd
+__fhopen40(fhp buffer[in], fh_size intptr, f flags[fhopen_flags]) fd
+compat_30_fhstat(fhp ptr[in, compat_30_fhandle], sb ptr[out, stat13])
+compat_30___fhstat30(fhp ptr[in, compat_30_fhandle], sb ptr[out, stat30])
+__fhstat50(fhp buffer[in], fh_size intptr, sb ptr[out, stat])
+compat_30_fhstatvfs1(fhp ptr[in, compat_30_fhandle], buf ptr[out, statvfs90], f flags[vfs_flags])
+compat_90_fhstatvfs1(fhp buffer[in], fh_size intptr, buf ptr[out, statvfs90], f flags[vfs_flags])
+__fhstatvfs190(fhp buffer[in], fh_size intptr, buf ptr[out, statvfs], f flags[vfs_flags])
```

Comment:
maybe we could use the len[] field here for the size? You seem to have used it in the other files for buffer size associations, any reason for not using them in this file?

Reply:
Yep, would include len[] in the description.